curated_transformers/util/hf.py

```diff
@@ -12,6 +12,7 @@
 HF_MODEL_SHARDED_CHECKPOINT_INDEX = "pytorch_model.bin.index.json"
 HF_MODEL_SHARDED_CHECKPOINT_INDEX_WEIGHTS_KEY = "weight_map"
 HF_TOKENIZER_CONFIG = "tokenizer_config.json"
+SPECIAL_TOKENS_MAP = "special_tokens_map.json"
 TOKENIZER_JSON = "tokenizer.json"
```

Comment:
Is it intentional that this global doesn't have a `HF_` prefix?

Reply:
Yeah, I think we should probably get rid of these prefixes (`TOKENIZER_JSON` also doesn't have this prefix). Since these live in the `hf` module now, they are already properly namespaced.